src/Ydb.php

```diff
@@ -107,7 +107,11 @@ public function __construct($config = [], LoggerInterface $logger = null)
         $this->database = $config['database'] ?? null;
         $this->iam_config = $config['iam_config'] ?? [];
 
-        if ($logger){
+        if (!is_null($logger) && isset($config['logger'])){
+            throw new \Exception('Logger set in 2 places');
+        } else if (isset($config['logger']) && $config['logger'] instanceof LoggerInterface) {
```

Comment:
no need check instanceof for logger in config in the place.

if logger present in config, but with bad type - SDK must fail instead of use NullLogger